docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+################################
+# OWT WebRTC server
+#
+# Base image Ubuntu 18.04
+
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+ENV MEDIA_SDK_VER=18.4.0
+
+# Update the system
+RUN apt update
+
+# Install utils
+RUN apt install -y ca-certificates git lsb-release mongodb nodejs npm sudo wget
+
+RUN npm install -g grunt-cli node-gyp
+
+# Keep proxy environment for sudo
+RUN bash -c "echo \"Defaults env_keep = \\\"http_proxy https_proxy no_proxy \\\"\" >> /etc/sudoers"
+
+# Install owt-client
+RUN git clone https://github.com/open-webrtc-toolkit/owt-client-javascript.git
+RUN cd owt-client-javascript/scripts && npm install && grunt
+ENV CLIENT_SAMPLE_PATH=/owt-client-javascript/dist/samples/conference
+
+RUN wget https://github.com/Intel-Media-SDK/MediaSDK/releases/download/intel-mediasdk-$MEDIA_SDK_VER/MediaStack.tar.gz -P /tmp
+RUN cd /tmp && tar -zxf MediaStack.tar.gz && cd MediaStack && ./install_media.sh
+RUN rm /tmp/MediaStack.tar.gz && rm -rf /tmp/MediaStack
+ENV MFX_HOME=/opt/intel/mediasdk
+
+RUN git clone https://github.com/open-webrtc-toolkit/owt-server.git
+WORKDIR /owt-server
+
+# This is needed to patch licode
+RUN  git config --global user.email "you@example.com" && \
+  git config --global user.name "Your Name"
+
+RUN ./scripts/installDepsUnattended.sh
+RUN ./scripts/build.js -t all --check
+RUN ./scripts/pack.js -t all --install-module --sample-path $CLIENT_SAMPLE_PATH .
```

Comment:
Please remove build.js and pack.js in the Dockerfile. These 2 scripts will be frequently executed for future MCU developers, so it's better to wrap them in a script and then developers can execute the script to finish build and pack process in the container. We should provide a clean MCU build environment as base docker image.

Reply:
@qwu16, maybe those 2 lines should be in another Dockerfile for the app itself, and this Dockerfile will be just the server.  Then docker-compose would pick the app container instead.

Reply:
I moved the build.js and pack.js steps into a separate Dockerfile, the main Dockerfile just builds the server, and the docker-compose.yml would build both of them.

Reply:
LGTM